spec/routing/bulletins_routing_spec.rb

```diff
@@ -1,14 +1,12 @@
 require 'rails_helper'
 
 describe 'routes for bulletins', type: :routing do
-  routes { Basechurch::Engine.routes }
-
   describe 'GET /v1/bulletins/:id' do
     let(:id) { 1 }
     let(:show_bulletin) { get "/v1/bulletins/#{id}" }
 
     it 'routes to bulletins#show(id)' do
-      expect(show_bulletin).to route_to(controller: 'basechurch/v1/bulletins',
+      expect(show_bulletin).to route_to(controller: 'v1/bulletins',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.